extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -0,0 +1,39 @@
+import Context from 'ad-engine/src/services/context-service';
+import SlotService from 'ad-engine/src/services/slot-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
+import config from './context';
+
+let blbSlot = {
+	config: {
+		targeting: {}
+	},
+	getSlotName: () => 'BOTTOM_LEADERBOARD',
+	getId: () => 'BOTTOM_LEADERBOARD'
+};
+
+Context.extend(config);
+TemplateService.register(BigFancyAdBelow);
+SlotService.add(blbSlot);
+
+function loadCustomAd(callback) {
+	return (params) => {
+		try {
+			const slot = SlotService.getBySlotName(params.slotName);
+			TemplateService.init(params.type, slot, params);
+		} catch (e) {
+			callback(params);
+		}
+	};
+}
+
+function updatePageLevelTargeting(params) {
+	Object.keys(params).forEach((key) => Context.set(`targeting.${key}`, params[key]));
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Replace this ES6 template string with string concatenation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AStringInterpolation)

[project-id]: wikia:app
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Convert this ES6 arrow function into a regular one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AArrowFunctions)

[project-id]: wikia:app